test/test_tensor.py

```diff
@@ -1030,41 +960,13 @@ def test_tensor_stack(n, shape, dim):
 
 @pytest.mark.parametrize("output", [Bint[2], Real, Reals[4], Reals[2, 3]], ids=str)
 def test_funsor_stack(output):
-    x = random_tensor(
-        OrderedDict(
-            [
-                ("i", Bint[2]),
-            ]
-        ),
-        output,
-    )
-    y = random_tensor(
-        OrderedDict(
-            [
-                ("j", Bint[3]),
-            ]
-        ),
-        output,
-    )
-    z = random_tensor(
-        OrderedDict(
-            [
-                ("i", Bint[2]),
-                ("k", Bint[4]),
-            ]
-        ),
-        output,
-    )
+    x = random_tensor(OrderedDict(i=Bint[2]), output)
+    y = random_tensor(OrderedDict(j=Bint[3]), output)
+    z = random_tensor(OrderedDict(i=Bint[2], k=Bint[4]), output)
```

Comment:
I like this one-line pattern when the expression is short. But when we add a comma `,` before the closing bracket `]` of a list (or a tuple), somehow black will expand it into multiple lines. A solution is to remove those extra `comma` for short expressions. What do you think? I can make a quick PR for it (using a hack: running `black` master with `--line-length 1000` and  `--skip-magic-trailing-comma` option to remove the final comma, then run `black` again with default options) so you can see what's the difference.

Reply:
Nevermind, the trick I mentioned above works well for blocks without "comments". For those with comments, the comments are moved to unexpected positions... ==